services/preview/vmware/mgmt/2019-08-09-preview/vmware/client.go

```diff
@@ -0,0 +1,52 @@
+// Package vmware implements the Azure ARM Vmware service API version 2019-08-09-preview.
+//
+// Azure VMware Solution API
+package vmware
```

Comment:
The package should be `avs`. Please see https://github.com/Azure/sdk-release-request/issues/496 for more details.